integration/setup/setup.go

```diff
@@ -7,6 +7,11 @@ import (
 	"os"
 	"testing"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/giantswarm/apprclient"
+	awsclient "github.com/giantswarm/aws-operator/integration/client"
+	awstemplate "github.com/giantswarm/aws-operator/integration/template"
```

Comment:
I don't think it would affect build times, i've taken care of not pulling the whole think. IMO refactoring this to a separate/multiple separate projects makes sense.

Reply:
@xh3b4sd done, updated to depend on e2eclients and e2etemplates instead of aws-operator, PTAL.